hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningServiceTest.java

```diff
@@ -231,6 +237,63 @@ public void run()
         }, 2);
     }
 
+
+    @Test
+    public void testExecutionShouldNotTimeoutIfTraceable_WithMultiCallback() throws Exception {
```

Comment:
Ah, now I get want you meant, sorry for the confusion. The test indeed seems to sleep for 15 seconds to trigger the operation timeout of 3 seconds (the callable result is checked, so the full time will elapse).

That one might indeed be a good candidate for a slow test.